src/fsharp/azure-pipelines-PR.yml

```diff
@@ -339,7 +339,7 @@ stages:
             continueOnError: true
           - task: PublishBuildArtifacts@1
             displayName: Publish Build BinLog
-            condition: eq(variables['System.JobPositionInPhase'], 1)
+            condition: and(succeeded(), eq(variables['System.JobPositionInPhase'], 1))
```

Comment:
I think this would change the behavior, right now we still publish build logs if the PublishTestResults fails (which has `continueOnError: true`)

Reply:
I think for cases where we want them to run always we could be explicit with either `succeededOrFailed()` or `always()` what do you think?

Also, feel free just apply edits to the branch as you see fit.

Reply:
yep